specification/specs/cluster/reload_secure_settings/ReloadSecureSettingsRequest.ts

```diff
@@ -25,5 +25,7 @@ class ReloadSecureSettingsRequest extends RequestBase {
   query_parameters?: {
     timeout?: Time
   }
-  body?: {}
+  body?: {
+    secure_settings_password: string
```

Comment:
This should be optional :)
```suggestion
    secure_settings_password?: string
```